packages/sdk/src/types/ValidationRules.ts

```diff
@@ -1,62 +1,220 @@
+}
+
+export interface ValidationRulesObject {
+	//13_obj_validation_rule_object
 	id: string
 	name: string
+	rules?: RulesObject
 	error?: {
 		message?: string
 	}
-	rules?: {
-		logic?: string
-		[rule: string]: any
-	}
+	applicable_to?: ApplicableToObject
 	created_at: string
 	updated_at?: string
+	type?: ValidationRulesObjectType
+	context_type?: ValidationRulesObjectContextType
+
 	object: 'validation_rules'
 }
 
-export interface ValidationRulesValidateResponse {
-	valid: boolean
-	rule_id: string
-	applicable_to: ApplicableToResultList
-}
+type ValidationRulesObjectType = 'basic' | 'advanced' | 'complex'
 
-export type ValidationRulesGetResponse = ValidationRulesCreateResponse & { assignments_count?: string }
+type ValidationRulesObjectContextType =
+	| 'earning_rule.order.paid'
+	| 'earning_rule.custom_event'
+	| 'earning_rule.customer.segment.entered'
+	| 'campaign.discount_coupons'
+	| 'campaign.discount_coupons.discount.apply_to_order'
+	| 'campaign.discount_coupons.discount.apply_to_items'
+	| 'campaign.discount_coupons.discount.apply_to_items_proportionally'
+	| 'campaign.discount_coupons.discount.apply_to_items_proportionally_by_quantity'
+	| 'campaign.discount_coupons.discount.fixed.apply_to_items'
+	| 'campaign.gift_vouchers'
+	| 'campaign.gift_vouchers.gift.apply_to_order'
+	| 'campaign.gift_vouchers.gift.apply_to_items'
+	| 'campaign.referral_program'
+	| 'campaign.referral_program.discount.apply_to_order'
+	| 'campaign.referral_program.discount.apply_to_items'
+	| 'campaign.referral_program.discount.apply_to_items_proportionally'
+	| 'campaign.referral_program.discount.apply_to_items_proportionally_by_quantity'
+	| 'campaign.referral_program.discount.fixed.apply_to_items'
+	| 'campaign.promotion'
+	| 'campaign.promotion.discount.apply_to_order'
+	| 'campaign.promotion.discount.apply_to_items'
+	| 'campaign.promotion.discount.apply_to_items_proportionally'
+	| 'campaign.promotion.discount.apply_to_items_proportionally_by_quantity'
+	| 'campaign.promotion.discount.fixed.apply_to_items'
+	| 'campaign.loyalty_program'
+	| 'campaign.lucky_draw'
+	| 'voucher.discount_voucher'
+	| 'voucher.discount_voucher.discount.apply_to_order'
+	| 'voucher.discount_voucher.discount.apply_to_items'
+	| 'voucher.discount_voucher.discount.apply_to_items_proportionally'
+	| 'voucher.discount_voucher.discount.apply_to_items_proportionally_by_quantity'
+	| 'voucher.discount_voucher.discount.fixed.apply_to_items'
+	| 'voucher.gift_voucher'
+	| 'voucher.gift_voucher.gift.apply_to_order'
+	| 'voucher.gift_voucher.gift.apply_to_items'
+	| 'voucher.loyalty_card'
+	| 'voucher.lucky_draw_code'
+	| 'distribution.custom_event'
+	| 'reward_assignment.pay_with_points'
+	| 'global'
 
 export type ValidationRulesUpdate = Partial<ValidationRulesCreate> & { id: string }
 
-export type ValidationRulesUpdateResponse = ValidationRulesCreateResponse
+export type ValidationRulesUpdateResponse = ValidationRulesObject
 
-export interface ValidationRulesCreateAssignment {
```

Comment:
Interface vs Type - please look at origin comment about the `DistributionsPublicationsCreateParams`